test/system/libs/helpers.bash

```diff
@@ -11,6 +11,11 @@ readonly SKOPEO=$(command -v skopeo)
 readonly PROJECT_DIR=${PWD}
 readonly IMAGE_CACHE_DIR="${PROJECT_DIR}/image-cache"
 
+readonly TOOLBOX_SYSTEM_CONFIG="/etc/containers/toolbox.conf"
+readonly TOOLBOX_USER_CONFIG="~/.config/containers/toolbox.conf"
+readonly BACKUP_SYSTEM_CONFIG="$BATS_TMPDIR/system-toolbox.conf"
+readonly BACKUP_USER_CONFIG="$BATS_TMPDIR/user-toolbox.conf"
```

Comment:
Are we really moving around the actual system-wide and user-specific configuration files for the test suite? If so, then it would make it difficult to isolate the test suite and make it non-destructive.

We should probably override `$HOME` for the user-specific one. If it's hard to redirect the system-wide file, then we could skip it.

Reply:
Maybe we should offer an env var for the tests switching on the tests working with system-wide configuration files.